### DISCUSSION TITLE
genuine question: why is useSteamOSConfig enabled by default

### DISCUSSION THREAD

**Discussion by nyawox** [Jovian-Experiments/Jovian-NixOS#428] [Q&A] [answered]
I appreciate the time and effort put in this project and I have no intention of offering criticism. however I felt odd that `useSteamOSConfig` is enabled by default. I get that this project was probably originally intended for use on steam deck rather than a traditional desktop, but I imagine that many nixos users may have a flake with multiple (>3) machines. and importing the nixos module might cause unexpected issues, such as portrait screen orientation, until they discover that `default` is set to `true;`

**Comment by samueldr** [comment, marked as answer]
Genuine answer: oops.

 - https://github.com/Jovian-Experiments/Jovian-NixOS/pull/430

**Reply by samueldr** [reply]
More answering:

It should have been. I believe modules that composes on top of NixOS should strive to be a no-op. Doing otherwise makes things *awkward* to reason about.

I can't quickly find the PR, but I went through the (much harder) ordeal with Mobile NixOS. So yeah, my opinion was already set :),

**Reply by nyawox** [reply]
haha that happens, it's very hard to learn good practices let alone executing it. thank you for addressing, and `check-no-ops.nix` was interesting to read
